tests/components/ring/test_sensor.py

```diff
@@ -1,122 +1,46 @@
 
-    def add_entities(self, devices, action):
-        """Mock add devices."""
-        for device in devices:
-            self.DEVICES.append(device)
+    front_door_last_activity_state = hass.states.get("sensor.front_door_last_activity")
+    assert front_door_last_activity_state is not None
 
-    def setUp(self):
-        """Initialize values for this testcase class."""
-        self.hass = get_test_home_assistant()
-        self.config = {
-            "username": "foo",
-            "password": "bar",
-            "monitored_conditions": [
-                "battery",
-                "last_activity",
-                "last_ding",
-                "last_motion",
-                "volume",
-                "wifi_signal_category",
-                "wifi_signal_strength",
-            ],
-        }
+    downstairs_wifi_signal_strength_state = hass.states.get(
+        "sensor.downstairs_wifi_signal_strength"
+    )
 
-    def tearDown(self):
-        """Stop everything that was started."""
-        self.hass.stop()
+    if not WIFI_ENABLED:
+        return
 
-    @requests_mock.Mocker()
-    def test_sensor(self, mock):
-        """Test the Ring sensor class and methods."""
-        mock.post(
-            "https://oauth.ring.com/oauth/token", text=load_fixture("ring_oauth.json")
-        )
-        mock.post(
-            "https://api.ring.com/clients_api/session",
-            text=load_fixture("ring_session.json"),
-        )
-        mock.get(
-            "https://api.ring.com/clients_api/ring_devices",
-            text=load_fixture("ring_devices.json"),
-        )
-        mock.get(
-            "https://api.ring.com/clients_api/doorbots/987652/history",
-            text=load_fixture("ring_doorbots.json"),
-        )
-        mock.get(
-            "https://api.ring.com/clients_api/doorbots/987652/health",
-            text=load_fixture("ring_doorboot_health_attrs.json"),
-        )
-        mock.get(
-            "https://api.ring.com/clients_api/chimes/999999/health",
-            text=load_fixture("ring_chime_health_attrs.json"),
-        )
+    assert downstairs_wifi_signal_strength_state is not None
+    assert downstairs_wifi_signal_strength_state.state == "-39"
 
-        with mock_storage(), patch("homeassistant.components.ring.PLATFORMS", []):
-            run_coroutine_threadsafe(
-                base_ring.async_setup(self.hass, VALID_CONFIG), self.hass.loop
-            ).result()
-            run_coroutine_threadsafe(
-                self.hass.async_block_till_done(), self.hass.loop
-            ).result()
-            run_coroutine_threadsafe(
-                ring.async_setup_entry(self.hass, None, self.add_entities),
-                self.hass.loop,
-            ).result()
+    front_door_wifi_signal_category_state = hass.states.get(
+        "sensor.front_door_wifi_signal_category"
+    )
+    assert front_door_wifi_signal_category_state is not None
+    assert front_door_wifi_signal_category_state.state == "good"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4821145)